pkg/csi/service/node.go

```diff
@@ -473,12 +474,17 @@ func (driver *vsphereCSIDriver) NodeUnpublishVolume(
 	}
 
 	// Figure out if the target path is a file or block volume.
-	isFileMount, _ := common.IsFileVolumeMount(ctx, target, mnts)
+	isFileMount, err := common.IsFileVolumeMount(ctx, target, mnts)
```

Comment:
The err was skipped here because the only time `IsFileVolumeMount` errors out is when target path is not available in mount points which has already been checked right before this function call.